src/bdfsclient/ActionHandler.cpp

```diff
@@ -111,22 +160,19 @@ namespace dfs
     // Cache size set to 100MB / (blockSize * (dataCount + codeCount)), flushing every 10 seconds
     volume->EnableCache(std::make_unique<dfs::Cache>("cache", volume.get(), 200, 10));
```

Comment:
I think we should make this into settings from optional argument or config files so that people can change the location and size. We could use the current value as defaults.